Working spawner with enemy follow, player child object facing mouse, etc

Ch-i-ef opened the pull request:
*Placeholder sprites for gun and bullet objects rn

Ch-i-ef committed:
Working spawner with enemy follow, player child object facing mouse, etc

*Placeholder sprites for gun and bullet objects rn

rafaVls committed:
clean up the Player script

the rb variable in line 8 doesn't need to be public, because you're getting it from the Player itself in line 14

rafaVls committed:
add missing types to FaceMouse

dir is a Vector3 and angle is a float

rafaVls committed:
add comments to FaceMouse

also add xAxis variable for clarity

rafaVls committed:
move distanceFromPlayer to its own variable

this way, it won't be calculated each frame (I think)

rafaVls committed:
fix indentation for else statement

it doesn't really make a difference functionality-wise, but it's more readable

rafaVls committed:
add spawnSpots to an array automatically

this way, you don't need to drag and drop each spawnSpot individually in Unity

rafaVls committed:
remove private declaration for playerPos

if I'm not wrong, declarations are private by default, so we don't really need to say it here. we can add it back if you think it's more clear that way

rafaVls committed:
move playerPos to its own variable in Update()

this is just for readability

rafaVls commented:
I added some commits to this PR. We can look at them later and decide what to get rid of.

I also opened issue #4, which we can also take a look at later.

rafaVls committed:
add UserSettings folder

this is a file that changes everytime I open the project in my computer, so maybe it's useful to have it

rafaVls committed:
fix spawner gameObject spawning enemies

this fixes #4 (hopefully)
